EDMCLogging.py

```diff
@@ -78,6 +80,33 @@ def get_logger(self) -> logging.Logger:
         return self.logger
 
 
+def get_plugin_logger(name: str, loglevel: int = _default_loglevel) -> logging.Logger:
+    """
+    'Found' plugins need their own logger to call out where the logging is
+    coming from, but we don't need to set up *everything* for them.
+
+    The name will be '{config.appname}.{plugin.name}', e.g.
+    'EDMarketConnector.miggytest'.  This means that any logging sent through
+    there *also* goes to the channels defined in the 'EDMarketConnector'
+    logger, so we can let that take care of the formatting.
```

Comment:
Will be or should be? We dont set the loggers name here, only use what we're given

Reply:
We set it in plug.py.  Either it's EDMarketConnector.<something> from that else it's bare EDMarketConnector due to being nothing to do with plugins.  The docs also now say to achieve the former *now* in plugin code.

The point is that we don't need to add a channel there because the message will propagate upwards to the EDMarketConnector logger which has one.

If a plugin author sets up a Logger with a name NOT of that format they're to blame when it doesn't log how we expect.

Reply:
Ah okay. Was mostly looking for the commentary of the name format being described here when it isnt set or enforced here. But if you're okay with "go against this at your own peril" that works for me